app/utils/store.py

```diff
@@ -66,17 +63,13 @@ def get(self, service_id, document_id, decryption_key):
                 Bucket=self.bucket,
                 Key=self.get_document_key(service_id, document_id),
                 SSECustomerKey=decryption_key,
-                SSECustomerAlgorithm='AES256'
+                SSECustomerAlgorithm="AES256",
             )
 
         except BotoClientError as e:
-            raise DocumentStoreError(e.response['Error'])
+            raise DocumentStoreError(e.response["Error"])
 
-        return {
-            'body': document['Body'],
-            'mimetype': document['ContentType'],
-            'size': document['ContentLength']
-        }
+        return {"body": document["Body"], "mimetype": document["ContentType"], "size": document["ContentLength"]}
```

Comment:
This kind of thing is clearly worse and less readable, but I guess that’s the price you pay for absolute consistency with no human judgment 🙃

Reply:
Fair, this is marginally worse, I agree.